packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.jsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const getTextForStake = (amount_of_stake, className, currency, text) => (
```

Comment:
```suggestion
const TextStake = ({ amount_of_stake, className, currency, text }) => (
```
This should be a component.

Reply:
I'll rebuild it in component